docs/variants/protectli_vp46xx/hardware-matrix.md

```diff
@@ -54,6 +54,30 @@ coreboot port on the Protectli VP46XX firewall.
 | **Power Control**              | RTE v1.1.0                                  |
 | **Power supply**               | Channel Well Technology 12V, 7.5A 90W       |
 
+## Protectli VP4650 ver2
+
+| Component                      | Description                                 |
+|--------------------------------|---------------------------------------------|
+| **CPU**                        | Intel Core i5-10210U                        |
+| **RAM**                        | Slot 1: Kingston 99U5624-013.A00G 8GB       |
+|                                | Slot 2: Kingston 99U5624-013.A00G 8GB       |
+| **Flash memory**               | Macronix KH25L12835F                        |
```

Comment:
```suggestion
| **Flash memory**               | KHIC KH25L12835F                            |
```

To be precise these chips are not manufactured by Macronix, but KHIC. Despite they are just copies of Macronix, we should document the original manufacturer, because Macronix does not produce KH25L12835F parts. Please change every occurrence where applicable.

Reply:
@miczyg1 fixed in 25909da883ac48247bf6a8374a9f4e3d3fe85041